instrumentation/opentelemetry-instrumentation-threading/tests/test_threading.py

```diff
@@ -13,20 +13,27 @@
 # limitations under the License.
 
 import threading
-from concurrent import futures
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from dataclasses import dataclass
+from typing import List
 
 from opentelemetry import trace
 from opentelemetry.instrumentation.threading import ThreadingInstrumentor
 from opentelemetry.test.test_base import TestBase
 
 
+@dataclass
+class MockContext:
+    span_context: trace.SpanContext = None
```

Comment:
I don't think this class is needed after all. The SpanContext alreadyt contains the trace and span IDs, so you don't need to record them separately.

Reply:
I used the `self._mock_span_contexts: List[trace.SpanContext] = []` to simplify mock class.